urid/src/feature.rs

```diff
@@ -1,85 +1,103 @@
 //! Thin but safe wrappers for the URID mapping features.
+
 use crate::{URIDCache, URID};
 use core::feature::Feature;
 use core::UriBound;
 use std::ffi::CStr;
+use std::marker::PhantomData;
 use std::os::raw::c_char;
 
 /// Host feature to map URIs to integers
 pub struct Map<'a> {
-    internal: &'a sys::LV2_URID_Map,
+    internal: sys::LV2_URID_Map,
+    _mapper_lifetime: PhantomData<&'a ()>,
 }
 
 unsafe impl<'a> UriBound for Map<'a> {
     const URI: &'static [u8] = sys::LV2_URID_MAP_URI;
 }
 
-impl<'a> Feature<'a> for Map<'a> {
-    type RawDataType = sys::LV2_URID_Map;
+unsafe impl<'a> Feature<'a> for Map<'a> {}
```

Comment:
You don't need the `'a` lifetime parameter in `Map` to implement `Feature` for it. Assuming you removed the lifetime parameter from `Map<'a>`, you could also write:
```suggestion
unsafe impl<'a> Feature<'a> for Map {}
```

Reply:
The lifetime parameter here is needed however (both for hosts and plugins): as the Map/Unmap do not simply provide pointers to free-standing functions like some other features, they need to be tied to the internal mapper, which is what this lifetime represents.